kindversions.properties

```diff
@@ -77,6 +77,7 @@
 0.16.0_1.20.15=kindest/node:v1.20.15@sha256:45d0194a8069c46483a0e509088ab9249302af561ebee76a1281a1f08ecb4ed3
 0.17.0_1.25=kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
 0.17.0_1.25.3=kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
+0.17.0_1.25.4=kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
```

Comment:
A little hacky ;) but okay. I notice that there is no Kind image for 1.25.4: https://hub.docker.com/r/kindest/node/tags?page=1